hack/fake-apiserver/README.md

```diff
@@ -0,0 +1,46 @@
+# CAPM3 Fake API server
+
+Fake API server is a tool running inside a kubernetes cluster,
+and "generates" in-memory k8s api server endpoints on demand.
+These endpoints can be used, for e.g., to represent the clusters
+created by CAPM3 using fake hardware so that CAPI confirms that
+the cluster was provisioned successfully.
+
+## How to use
+
+After building the container image and deploy it to a kubernetes cluster,
```

Comment:
A minimal example of how to do this would be very helpful.
```bash
docker build ...
kind load image ... # or docker push
kubectl create deploy ...
kubectl create service ... # or get IP of pod
kubectl port-forward ...
```